LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
+
+Copyright (c) 2017, Science and Technology Failities Council
```

Comment:
Unfortunately you too have got a typo in your license: "Failities" and, just like mine, it's in every file :-(